framework/src/operators/mod.rs

```diff
@@ -21,10 +22,13 @@ mod map_batch;
 mod queue_batch;
 mod receive_batch;
 mod send_batch;
+mod complete_batch;
 
 /// Error when processing packets
 #[derive(Debug)]
 pub enum PacketError {
+    /// Processing is complete; not really an error but we can short-circuit processing
```

Comment:
Not averse to rename `PacketError` to something more appropriate. `Drop` is not an error condition either. Maybe `PacketOutcome`? `PacketDisposition`? all sound kinda weird, that's why I went with `PacketError` even though it's really kinda wrong.

Reply:
Yeah, `PacketResult`? but `Result<T, PacketResult>` is maybe the eye-bleedingest. I could probably get behind `PacketOutcome`.